packages/vaadin-combo-box/src/vaadin-combo-box-dropdown-wrapper.js

```diff
@@ -458,11 +425,6 @@ class ComboBoxDropdownWrapperElement extends PolymerElement {
     });
   }
 
-  updateViewportBoundaries() {
-    this._cachedViewportTotalPaddingBottom = undefined;
-    this._selector.updateViewportBoundaries();
-  }
-
   get _viewportTotalPaddingBottom() {
     if (this._cachedViewportTotalPaddingBottom === undefined) {
       const itemsStyle = window.getComputedStyle(this._selector.$.items);
```

Comment:
This does not exist anymore: `this._selector.$.items` as it was used by [`iron-list`](https://github.com/PolymerElements/iron-list/blob/f870df1fcbf8f85c4b17c3c35b19793c0c0bbe20/iron-list.js#L287) internally.

Also this whole getter seems no longer used and can be removed.

Reply:
Done.